pallets/oracle-benchmarks/Cargo.toml

```diff
@@ -51,6 +51,7 @@ runtime-benchmarks = [
 	"sp-runtime/runtime-benchmarks",
 	"astar-primitives/runtime-benchmarks",
 	"frame-benchmarking/runtime-benchmarks",
+	"orml-oracle/runtime-benchmarks",
```

Comment:
will check. zepter added this as missing :)

Reply:
Not doubting it's missing since we uplifted, but `oracle-benchmarks` (not `orml-oracle`) should be obsolete now.

Might also be neat to re-run the benchmarks for the oracle, just to make sure everything works.
With that being said, it might be a prudent thing to do for everything given we've uplifted "a lot" of versions now.